packages/fscomponents/src/components/ReviewsSummary.tsx

```diff
@@ -72,7 +72,7 @@ export class ReviewsSummary extends Component<ReviewsSummaryProps> {
         <View style={[S.row, rowStyle]}>
           <Text style={[S.recommendStyle, recommendStyle]}>
             {FSI18n.string(translationKeys.flagship.reviews.recommendCount, {
-              recommendPercent: FSI18n.percent(recommend)
```

Comment:
Does this need to be removed? The reference project is using a version of this file with FSI18n.percent() still used.